packages/mobile/src/components/ContactCircle.tsx

```diff
@@ -41,7 +41,10 @@ function ContactCircle({ size, recipient, style }: Props) {
     if (recipient.name) {
       const initials = getNameInitial(recipient.name)
       return (
-        <Text style={[fontStyles.iconText, { fontSize: iconSize / 2.0, color: fontColor }]}>
+        <Text
+          allowFontScaling={false}
```

Comment:
if we turn off font scaling doesn't that make accessibility basically not work anymore? not sure how this works so just thinking out loud...

Reply:
It does prevent the font from sizing up. This specific bit of code is responsible for the initial seen in the contact circle, as such resizing the text was escaping the boundaries of the contact circle. I tried scaling up the circle, but it caused layout shifts. To keep the icon a circle I opt'd to disallow font scaling.